examples/connext_dds/group_coherent_presentation/c++11/group_coherent_subscriber.cxx

```diff
@@ -0,0 +1,205 @@
+                      std::cout << sample << std::endl;
+                      print_coherent_set_info(sample.info());
+                  }
+              } else if ((*it).topic_name() == "Checkpoint Place") {
+                  dds::sub::LoanedSamples<CheckpointPlace> samples = 
+                      (*it).get<CheckpointPlace>().take();
+                  for (const rti::sub::LoanedSample<CheckpointPlace>& sample : samples) {
+                      std::cout << sample << std::endl;
+                      print_coherent_set_info(sample.info());
+                  }
+              }
+          }
+    } // end coherent access
+} 
+
+void run_subscriber_application(unsigned int domain_id)
+{
+    dds::domain::DomainParticipant participant(domain_id);
+
+    dds::topic::Topic<CheckpointTime> time_topic(
+            participant, 
+            "Checkpoint Time");
+    dds::topic::Topic<CheckpointPlace> place_topic(
+            participant, 
+            "Checkpoint Place");
+
+    // Retrieve the Subscriber QoS, from USER_QOS_PROFILES.xml.
+    SubscriberQos subscriber_qos = QosProvider::Default().subscriber_qos();
+
+    // If you want to change the Subscriber's QoS programmatically rather
+    // than using the XML file, you will need to uncomment the following line.
+    // subscriber_qos << Presentation::GroupAccessScope(true, true);
+    // subscriber_qos.policy<Presentation>().extensions().drop_incomplete_coherent_set(false);
+
+    dds::sub::Subscriber subscriber(participant, subscriber_qos);  
+
+    // Retrieve the default DataReader QoS, from USER_QOS_PROFILES.xml
+    DataReaderQos reader_qos = QosProvider::Default().datareader_qos();
+
+    // If you want to change the DataReader QoS programmatically rather
+    // than using the XML file, you will need to comment out these lines.
+    // reader_qos << Reliability::Reliable()
+    //            << History::KeepAll();
+
+    auto time_listener =
+            std::make_shared < CheckpointDRListener<CheckpointTime>>();
+    auto place_listener =
+            std::make_shared < CheckpointDRListener<CheckpointPlace>>();
+
+    // We are installing a listener for the sample lost status in case an 
+    // incomplete coherent set is received and dropped (assuming the 
+    // PresentationQosPolicy::drop_incomplete_coherent_set is true (the default)
+    dds::sub::DataReader<CheckpointTime> time_reader(
+            subscriber, 
+            time_topic, 
+            reader_qos, 
+            time_listener, 
+            dds::core::status::StatusMask::sample_lost());
+    dds::sub::DataReader<CheckpointPlace> place_reader(
+            subscriber, 
+            place_topic, 
+            reader_qos, 
+            place_listener, 
+            dds::core::status::StatusMask::sample_lost());
+
+    // WaitSet will be woken when the attached condition is triggered
+    dds::core::cond::WaitSet waitset;
+
+    // Create a ReadCondition for any data on this reader, and add to WaitSet
```

Comment:
Update this comment (or get rid of it and the one on line 148)

Reply:
removed